src/models/Event.js

```diff
@@ -122,8 +125,13 @@ class Event extends Base {
   $beforeInsert () {
     const now = new Date().toISOString()
 
-    if (!this.createdDate) {
+    if (!this.createdDate && !this.createdTimestamp) {
       this.createdDate = now
+      this.createdTimestamp = now
+    } else if (!this.createdDate) {
+      this.createdDate = this.createdTimestamp
+    } else if (!this.createdTimestamp) {
+      this.createdTimestamp = this.createdDate
```

Comment:
Yes the conversation is automatic.

Good idea. Maybe not for the model event because we set the `createdDate` manually to match the targeted resource date.

Example for user creation event : https://github.com/stelace/stelace/blob/0aa12cc5066778bbd6cc893d9b8da497ce134451/src/services/user.js#L874-L879

 Any other model can benefit from the PostgreSQL function. I'll add this function.

Reply:
It’s not so useful if we don’t use it for Event that is the most important table for timescaleDB.
Moreover, lately you said on slack you probably wouldn’t use BRIN indexes, so why do you still want to create the function?

Reply:
That's right. Then there's no need to create this function.